README.md

```diff
@@ -28,10 +28,20 @@ $> git clone --recursive git@github.com:osclass/Osclass.git
 
 Go to [our site][installing] to get detailed information on installing Osclass.
 
+## Heroku
```

Comment:
`BUILDPACK_URL` is deprecated, you'll want to use `heroku buildpacks:set https://github.com/heroku/heroku-buildpack-php` instead.